### PR TITLE
Fixed minor issues in README and docs

### PR DESCRIPTION
**Pull request by andy-maier**
Link to README page as of this PR, for checking the picture: https://github.com/pywbem/pywbem/blob/andy/fix-readme/README.rst

Note there are also some updates to the docs - see commit message or Files changed.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -121,7 +121,7 @@ environment:
 
 This will also install any prerequisite Python packages.
 
-Since version 1.0.0, pywbem has no more OS-level prerequisite packages.
+Starting with version 1.0.0, pywbem has no more OS-level prerequisite packages.
```
Comment (KSchopmeyer) [inline README.rst:124]: Chang "no more" to "no"

File: docs/intro.rst [hunk 2]
```diff
@@ -122,7 +122,7 @@ operating system packages in addition to Python itself.
 
   This will also install any prerequisite Python packages.
 
-  Since version 1.0.0, pywbem has no more OS-level prerequisite packages.
+  Starting with version 1.0.0, pywbem has no more OS-level prerequisite packages.
```
Comment (KSchopmeyer) [inline docs/intro.rst:125]: Change "no more" to "no"

File: tests/unittest/pywbem/test_recorder.py [hunk 3]
```diff
@@ -2755,7 +2755,7 @@ class TestLoggingEndToEnd(BaseLogOperationRecorderTests):
     """
 
     def build_repo(self, namespace):
-        """Build a fake repo and FakeWBEMConnection so we get responses back"""
+        """Build a mock repo via FakedWBEMConnection so we get responses back"""
```
Comment (KSchopmeyer) [inline tests/unittest/pywbem/test_recorder.py:2758]: Change "via" to "using" or "with"

File: README.rst (hunk 1 above)
Comment (andy-maier) [reply README.rst:124]: DONE

File: docs/intro.rst (hunk 2 above)
Comment (andy-maier) [reply docs/intro.rst:125]: DONE

File: tests/unittest/pywbem/test_recorder.py (hunk 3 above)
Comment (andy-maier) [reply tests/unittest/pywbem/test_recorder.py:2758]: DONE

**Pull request merged**